CHANGES.md

```diff
@@ -1,11 +1,300 @@
+* made many changes, mostly minor, in support of Chapel 2.0 stabilization
+* many other bug fixes and improvements to error messages and documentation
+
+Packaging / Configuration Changes
+---------------------------------
+* the compiler's LLVM back-end now supports version 14 as well as 11, 12, 13
+* made Chapel available as an Arch Linux AUR package  
+  (see https://aur.archlinux.org/packages/chapel)
+* updated `configure` to create prefix directories if they don't already exist
+* `make install` now installs `chpldoc` if it is built
+
+Semantic Changes / Changes to Chapel Language
+---------------------------------------------
+* made small integers prefer coercions to `real(32)` over `real(64)`  
+  (see https://chapel-lang.org/docs/1.27/language/spec/conversions.html#implicit-numeric-and-bool-conversions)
+* made `public use` and `import` statements no longer introduce a hidden scope  
+  (see https://chapel-lang.org/docs/1.27/language/spec/modules.html#using-and-importing)
+* made `public use M` no longer bring in the name `M` for qualified access  
+  (see https://chapel-lang.org/docs/1.27/language/spec/modules.html#public-and-private-use-statements)
+* changed methods to no longer be subject to shadowing  
+  (see https://chapel-lang.org/docs/1.27/language/spec/procedures.html#determining-more-specific-functions)
+* made loops over unbounded enum/bool ranges stop at their extreme values
+* made serial loops over unbounded integer ranges terminate instead of spinning
+* byte-based string slicing now throws when not aligned to codepoint boundaries
+
+Deprecated / Unstable / Removed Language Features
+-------------------------------------------------
+* `.low`/`.high` on strided ranges now warn about pending behavioral changes  
+  (see https://chapel-lang.org/docs/1.27/language/spec/ranges.html#ChapelRange.range.low)
+* marked `.equals()` on arrays as being unstable
+* removed deprecated support for `.front()`/`.back()` on arrays
+* removed support for bracket-less anonymous associative domains
+
+New Features
+------------
+* added new `.lowBound`/`.highBound` queries on ranges to get "pure" bounds  
+  (e.g., `(1..10 by 2).highBound` returns `10`)  
+  (see https://chapel-lang.org/docs/1.27/language/spec/ranges.html#ChapelRange.range.lowBound)
+* added an `alignedBoundsByDefault` config to opt into new `.low`/`.high` defs  
+  (see https://chapel-lang.org/docs/1.27/language/spec/ranges.html#ChapelRange.alignedBoundsByDefault))
+
+Feature Improvements
+--------------------
+* `param` indexing of `string` and `bytes` values is now done at compile-time
+* changed one-tuples to print out as `(10,)` rather than `(10)` by default
+
+Namespace Changes
+-----------------
+* made internal module names not visible to Chapel programs by default
+
+Changes / Feature Improvements in Libraries
+-------------------------------------------
+* updated `.mod()`/`%` behavior for `bigint`s to match `mod()`/`%` for `int`s  
```

Comment:
`.mod()` -> `mod()`? or is this the syntax we use to denote methods?

Reply:
Yeah, that was meant to imply a method.  I would've used `bigint.mod()` if I hadn't been trying to describe both mod and % in a single bullet within the one-line constraint.

Reply:
(also, note that the integer mod is not a method, which is the reason for the asymmetry).